### PR TITLE
feat(gha): provide crds subchart as oci artifact

### PR DESCRIPTION
**Pull request by csatib02**
- Provide CRD's subchart as OCI artifact

Fixes: #1853

### CODE REVIEW

**Review by aslafy-z — APPROVED** [review]
LGTM! Thank you @csatib02

**Comment by csatib02** [comment]
Artifacts will be available at: <https://github.com/kube-logging/logging-operator/pkgs/container/helm-charts%2Flogging-operator%2Fcrds>

**Review by aslafy-z — APPROVED** [review]

File: .github/workflows/artifacts.yaml [hunk 1]
```diff
@@ -373,3 +382,126 @@ jobs:
...
+      security-events: write
+  
+    outputs:
+      name: ${{ steps.oci-chart-name.outputs.value }}
+      tag: ${{ steps.version.outputs.value }}
+      package: ${{ steps.build.outputs.package }}
+  
+    env:
+      subchartPath: logging-operator/charts/crds
+      subchartName: crds
```
Comment (OverOrion) [inline .github/workflows/artifacts.yaml:404]: [Nit]
I would rename it to `logging-operator-crds`, would it break anything?

**Review by OverOrion — APPROVED** [review]
Overall LGTM, I only have a minor remark regarding the subchart's name.

File: .github/workflows/artifacts.yaml (hunk 1 above)
Comment (csatib02) [reply .github/workflows/artifacts.yaml:404]: If you check the [uploaded artifacts](https://github.com/kube-logging/logging-operator/pkgs/container/helm-charts%2Flogging-operator%2Fcrds), it uses a full-path that contains the repo-name. `kube-logging/helm-charts/logging-operator/crds`

**Pull request merged**